docs/Scaling/overview.md

```diff
@@ -52,3 +52,11 @@ AKS is a container orchestration system that enables administrators to deploy an
     The AKS deployment section is not a tutorial for beginners. It intended for those who have prior knowledge of the service. To begin learning about AKS, start with [Azure Kubernetes Service](https://docs.microsoft.com/en-us/azure/aks/).
 
 [Get Started with AKS](../Kubernetes/deployingk8s-aks.md){: .md-button .md-button--primary target="_blank"}
+
+## Amazon Elastic Kubernetes Service (EKS)
+EKS is a container orchestration system that enables administrators to deploy and manage large numbers of containers and services. 
+
+!!! Warning
+    The EKS deployment section is not a tutorial for beginners. It intended for those who have prior knowledge of the service. To begin learning about EKS, start with [Amazon Elastic Kubernetes Service](https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html).
+
+[Get Started with EKS](../Kubernetes/deployingk8s-eks.md){: .md-button .md-button--primary target="_blank"}
```

Comment:
Lastly, I would just say try to be consistent ending complete sentences with periods unless they are a heading and using colons appropriately.